insights/client/core_collector.py

```diff
@@ -51,7 +51,11 @@ def run_collection(self, conf, rm_conf, branch_info, blacklist_report):
 
         manifest = collect.default_manifest
         if hasattr(self.config, 'manifest') and self.config.manifest:
-            manifest = self.config.manifest
+            if self.app is None:
```

Comment:
```suggestion
            if self.config.app is None:
```

Reply:
Oh duh. Thank you. I should not be allowed to write Python.